apps/desktop/src-tauri/tauri.conf.json

```diff
@@ -60,14 +60,13 @@
         "entitlements": null,
         "exceptionDomain": "",
         "frameworks": [
-          "frameworks/libonnxruntime.dylib"
+            "frameworks/libonnxruntime.dylib"
         ],
         "providerShortName": null,
         "signingIdentity": null
       },
       "resources": [
         "model/*",
-        "dylibs/*",
```

Comment:
don't we still need this for linux builds?

Reply:
have re-added this, will trigger another build.

Reply:
works after re-adding.